packages/web-components/src/components/ic-radio-option/ic-radio-option.tsx

```diff
@@ -37,7 +40,7 @@ export class RadioOption {
   /**
    * Provide a value for the input.
    */
-  @Prop() value!: string;
+  @Prop({ mutable: true }) value!: string;
```

Comment:
Does this need to be `mutable` as I cannot see where we are setting the value internally again?

Reply:
Was giving a warning in the unit tests, as the value is updated in handleClick method